### PR TITLE
cChunk::SetAlwaysTicked implies cChunk::stay

### PR DESCRIPTION
**Pull request by LogicParrot**
It makes more sense if always ticking chunks are always loaded too.

### CODE REVIEW

**Comment by LogicParrot** [comment]
Does anyone disagree with this change?

**Comment by tigerw** [comment]
Apparently not :P

File: src/Chunk.cpp [hunk 1]
```diff
@@ -1715,10 +1715,12 @@ void cChunk::SetAlwaysTicked(bool a_AlwaysTicked)
 	if (a_AlwaysTicked)
 	{
 		m_AlwaysTicked += 1;
+		Stay(a_AlwaysTicked);
```
Comment (tigerw) [inline src/Chunk.cpp]: So is m_AlwaysTicked now equivalent to m_StayCount? Or is there something preventing chunks from being ticked? I'm presuming the latter.

Comment (LogicParrot) [inline src/Chunk.cpp]: `m_AlwaysTicked` is always smaller or equal to `m_StayCount`. A chunk may have chunkStay without being always ticked.

Comment (LogicParrot) [inline src/Chunk.cpp]: In other words, the call to `Stay` may either be made independently, or implicitly through a call to `SetAlwaysTicked`. It's simply weird to want an always ticking but not always loaded chunk.

Comment (LogicParrot) [inline src/Chunk.cpp]: In other words, the call to `Stay` may either be made independently, or implicitly through a call to `SetAlwaysTicked`. It's simply weird to want an always ticking but not always loaded chunk. The other way around is still possible.

**Comment by madmaxoft** [comment]
Weird thing that this has broken the APIDump documentation and the plugin didn't complain in CI.
Fixed in #3400

**Comment by madmaxoft** [comment]
Oh, it failed to even load the APIDump plugin. But the CI won't mark that as an error. That needs to be fixed, too.

**Pull request merged**